### PR TITLE
Apis details 

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- add ApiDetails component
- add ApiDetailsHeader component
- display Api spec in details view
- unit tests
- use InProgressMessage component in OpenApi details page

### Api spec files:

[api_specs.zip](https://github.com/kyma-project/console/files/3555842/api_specs.zip)

**Related issue(s)**

### CODE REVIEW

File: compass/src/components/Api/ApiDetails/ApiDetails.component.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  if (
+    rawApisForApplication &&
+    rawApisForApplication.data &&
+    rawApisForApplication.data.length
+  ) {
+    const apisForApplication = rawApisForApplication.data;
+    const idToLookFor = apiId || eventApiId;
+
+    api = apisForApplication.find(a => a.id === idToLookFor);
```
Comment (Wawrzyn321) [inline compass/src/components/Api/ApiDetails/ApiDetails.component.js]: Would you kindly move finding the API to separate function?

File: compass/src/App.component.js [hunk 2]
```diff
@@ -64,6 +65,26 @@ class App extends React.Component {
               <ApplicationDetails applicationId={match.params.id} />
             )}
           />
+          <Route
+            path="/application/:id/api/:apiid"
```
Comment (Wawrzyn321) [inline compass/src/App.component.js]: _apiid_ -> _apiId_, and so on, to follow _applicationId_ and _runtimeId_.

File: compass/src/components/Api/ApiDetails/ApiDetails.component.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  deleteApi,
+  deleteEventApi,
+  apiId,
+  eventApiId,
+  applicationId,
+}) => {
+  const query = apiId ? getApisForApplication : getEventApisForApplication;
+
+  const { loading, error } = query;
+  const application = query && query.application;
```
Comment (Wawrzyn321) [inline compass/src/components/Api/ApiDetails/ApiDetails.component.js]: You could include the _application_ into deconstruction the line above.

File: compass/src/components/Api/ApiDetails/ApiDetailsHeader/ApiDetailsHeader.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                url="#"
+                onClick={navigateToApplication}
+              />
+              <BreadcrumbItem />
+            </Breadcrumb>
+            <ActionBar.Header title={this.props.api.name} />
+          </section>
+          <ActionBar.Actions>
+            <Button
+              onClick={() => this.handleDelete(this.props.api)}
```
Comment (Wawrzyn321) [inline compass/src/components/Api/ApiDetails/ApiDetailsHeader/ApiDetailsHeader.js]: We could use our brand-new _simpleDelete_ here.

File: compass/src/App.component.js (hunk 2 above)
Comment (parostatkiem) [reply compass/src/App.component.js]: Done

**Pull request merged**